Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas.git
-  revision: 5f18ff9ac519360826fda2f5373ffb8da2d8336c
-  tag: v0.1.6
+  revision: 5df43ce58a62d0c4fbbf5ce8521e4967e09c9e30
+  tag: v0.1.8
   specs:
-    laa-criminal-legal-aid-schemas (0.1.6)
+    laa-criminal-legal-aid-schemas (0.1.7)
```

Comment:
I think the schemas version was not bumped 😄

Reply:
I wonder if we could spec this somehow - if the hash of the schema files doesn't match the one in the spec we know it is out of date?

Reply:
What I was referring was the `version.rb` is still in 0.1.7 but the tagged version is 0.1.8.
To be fair this is perfectly valid, you can have tags whatever you want with the name you want even, it just we may want to keep consistency so both are in-sync 😛

Reply:
Yes, I think it happened when I rebased against Mo's change. Sorry! Will fix.

Reply:
No worries, can be fixed in next version (the one with enums). So both get in sync again.